### ISSUE TITLE
Example?

### ISSUE THREAD

**Issue by renderit** [fhussonnois/storm-cassandra#1]
Hey is it possible you can provide a complete example? I have a json document coming from another bolt. I'm trying to save that into cassandra db.
1. 
```
 new CassandraWriterBolt(
        insertInto("album")
            .values(
                with(fields("title", "year", "performer", "genre", "tracks")
                ).build());
``` 
goes into the execute method?

2. Where do I change the config properties? I don't have cassandra on localhost but on 3 another instances..

**Comment by fhussonnois** [comment]
Hi @renderit, actually this repository is not any more supported. This connector is now part of the Apache Storm distribution since version 1.0. This component is compatible with prior version.

https://github.com/apache/storm/tree/master/external/storm-cassandra

**Comment by renderit** [comment]
@fhussonnois Right, but I'm using storm 0.10.1 (not the latest 1.0); so the example there won't work I believe? (dependence on org.apache instead of backtype).. What should I be using. I'm in a hurry, possible I can still use this with cassandra 3.0.7?
